ci-operator/step-registry/vsphere-acm-virt/cold-migration/vsphere-acm-virt-cold-migration-commands.sh

```
@@ -0,0 +1,375 @@
+  namespace: ${MTV_NAMESPACE}
+spec:
+  plan:
+    name: vsphere-cold-plan
+    namespace: ${MTV_NAMESPACE}
+ocEOF
+
+# --------------------------------------------------------------------------
+# Wait for Migration to succeed
+# --------------------------------------------------------------------------
+typeset succeeded='' failed=''
+typeset -i deadline=$(( SECONDS + MIGRATION_TIMEOUT ))
```

Shellcheck (0.11.0)

[info] SECONDS was modified in a subshell. That change might be lost.

(SC2031)